tools/cli/preRelease.js

```diff
@@ -62,6 +62,18 @@ const updateVersionAndIntegrity = async (absolutePath, version, integrityHash) =
   fs.writeFileSync(absolutePath, result);
 };
 
+const updateChangeLog = async version => {
+  const relativePathToChangelog = '../../packages/teams-js/CHANGELOG.md';
+  const absolutePathToChangelog = path.resolve(__dirname, relativePathToChangelog);
+  if (!fs.existsSync(absolutePathToChangelog)) {
+    throw `ERROR: ${absolutePathToChangelog} was not found.`;
+  }
+  await execShellCommand('yarn beachball bump');
+  const changeLog = fs.readFileSync(absolutePathToChangelog, 'utf8');
+  const newChangeLog = changeLog.replace(/(## 2.0.0)/, `## ${version}"`);
```

Comment:
Is the double quote after ${version} intended?

Reply:
Great catch! Fixed